tests/Doctrine/Tests/Common/Cache/BaseFileCacheTest.php

```diff
@@ -31,6 +44,8 @@ public function tearDown()
                 @rmdir($file->getRealPath());
             }
         }
+
+        @rmdir($this->directory);
```

Comment:
the cleanup was missing to remove the cache root dir thus every test run would create a new temp dir without deleting it.